blockly-ws-search/src/WorkspaceSearch.js

```diff
@@ -463,10 +466,13 @@ export class WorkspaceSearch {
   /**
    * Returns whether the given block matches the search text.
    * @param {!Blockly.BlockSvg} block The block to check.
+   * @param {string} searchText The search text. Note if the search is case
+   *    insensitive, this will be passed already converted to lowercase letters.
```

Comment:
Not sure if I understand this comment, why will it be passed already converted to lowercase?

Reply:
Our setSearchText converts it to lowercase. I wanted to communicate that.

Reply:
sorry was reading case sensitive instead of case insensitive. Makes sense

Reply:
I've moved the logic for calling setLowerCase on searchTest  as discussed offline.